ui/__tests__/components/thing-counters.test.jsx

```diff
@@ -0,0 +1,46 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import ThingCounter from '../../components/thing-counters';
+import mockRouter from '../util/mockRouter';
+
+
+describe('<ThingCounter />', () => {
+  it('states zero results when there are no results', () => {
+    const context = { router: mockRouter() };
+    const result = shallow(<ThingCounter count={0} chunk={0} singular={ "requirement" } plural={ "requirements" }/>, { context });
+    expect(result.text()).toMatch(/No requirements match your search, try removing some filters to see more results./);
+    expect(result.find("div").length).toEqual(1);
```

Comment:
NBD, but there's a `expect(...).toHaveLength()` function, if you like that more.